<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="0" failed="0" ignored="0" total="1" passed="1">
  <reporter-output>
  </reporter-output>
  <suite name="Default suite" duration-ms="11699" started-at="2020-04-13T09:08:59 IST" finished-at="2020-04-13T09:09:11 IST">
    <groups>
    </groups>
    <test name="Default test" duration-ms="11699" started-at="2020-04-13T09:08:59 IST" finished-at="2020-04-13T09:09:11 IST">
      <class name="day9.ExceptionHandlingWithSelenium">
        <test-method status="PASS" signature="example()[pri:0, instance:day9.ExceptionHandlingWithSelenium@694e1548]" name="example" duration-ms="11649" started-at="2020-04-13T09:08:59 IST" finished-at="2020-04-13T09:09:11 IST">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- example -->
      </class> <!-- day9.ExceptionHandlingWithSelenium -->
    </test> <!-- Default test -->
  </suite> <!-- Default suite -->
</testng-results>
